src/core/config-schema.helper.ts

```diff
@@ -0,0 +1,125 @@
+import * as Joi from '@hapi/joi';
+import chalk from 'chalk';
+
+const schema = Joi.object({
+  browserWidth: Joi.number()
+    .min(100)
+    .max(2000)
+    .required(),
+  browserHeight: Joi.number()
+    .min(100)
+    .max(2000)
+    .required(),
+  browserLanguage: Joi.string().required(),
+  timeout: Joi.number()
+    .min(1)
+    .required(),
+  elementsVisibilityTimeout: Joi.number()
+    .min(1)
+    .required(),
+  waitForPageTimeout: Joi.number()
+    .min(1)
+    .required(),
+  downloadTimeout: Joi.number()
+    .min(1)
+    .required(),
+  reports: Joi.string().required(),
+  downloads: Joi.string().required(),
+  data: Joi.string().required(),
+  features: Joi.array()
+    .items(Joi.string())
+    .required(),
+  pages: Joi.array()
+    .items(Joi.string())
+    .required(),
+  matchers: Joi.array()
+    .items(Joi.string())
+    .required(),
+  generators: Joi.array()
+    .items(Joi.string())
+    .required(),
+  form_handlers: Joi.array()
+    .items(Joi.string())
+    .required(),
+  step_definitions: Joi.array()
+    .items(Joi.string())
+    .required(),
+  comparators: Joi.array()
+    .items(Joi.string())
+    .required(),
+  dictionaries: Joi.array()
+    .items(Joi.string())
+    .required(),
+  transformers: Joi.array()
+    .items(Joi.string())
+    .required(),
+  regexes: Joi.array()
+    .items(Joi.string())
+    .required(),
+  hooks: Joi.array()
+    .items(Joi.string())
+    .required(),
+  imageComparator: {
+    baselineFolder: Joi.string().required(),
+    temporaryFolder: Joi.string().required(),
+    saveAboveTolerance: Joi.number()
+      .min(1)
+      .max(99)
+      .required(),
+  },
+  clearEmailInboxBeforeTests: Joi.boolean().required(),
+  clearCookiesAfterScenario: Joi.boolean().required(),
+  clearLocalStorageAfterScenario: Joi.boolean().required(),
+  email: Joi.string()
```

Comment:
This is a configuration for email, not email with domain e.g. 'janek@email.com'

Reply:
Ok, I've deleted that field because it is optional.